src/utils/Database/LSMTree/Util/Orphans.hs

```diff
@@ -0,0 +1,72 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleInstances  #-}
+{-# LANGUAGE NamedFieldPuns     #-}
+{-# LANGUAGE StandaloneDeriving #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Database.LSMTree.Util.Orphans () where
+
+import           Control.DeepSeq (NFData (..))
+import qualified Data.Array.Unboxed as A
+import qualified Data.ByteString.Builder as B
+import qualified Data.ByteString.Lazy as LBS
+import qualified Data.ByteString.Short.Internal as SBS
+import           Data.Primitive.ByteArray (ByteArray (..))
+import           Data.WideWord.Word256 (Word256 (..))
+import           Data.Word (Word64)
+import           Database.LSMTree.Internal.Run.BloomFilter (Hashable (..))
+import           Database.LSMTree.Internal.Run.Index.Compact (CompactIndex (..))
+import           Database.LSMTree.Internal.Serialise (Serialise (..),
+                     SerialisedKey (..))
+import           GHC.Generics (Generic)
+import           System.Random (Uniform)
+
+deriving instance Generic SerialisedKey
+deriving instance NFData SerialisedKey
+
+deriving instance Generic CompactIndex
+deriving instance NFData CompactIndex
+
+instance NFData (A.UArray Int Bool) where
+  rnf x = rnf (A.bounds x, A.elems x)
```

Comment:
NF is the same as WHNF for `Data.Array.Unboxed.UArray`.
```suggestion
  rnf x = x `seq` ()
```

Reply:
Good point

I also realise now we're not using this `NFData` instance anywhere, so we can remove it